frontend/src/app/pages/marker-page/marker-page.component.html

```diff
@@ -38,6 +38,14 @@ <h3>Your Labels</h3>
 
         <div class="actions">
             <h3>Actions</h3>
+            <div class="buttons" *ngIf="this.labelExplorer.getLabelsLength() > 0">
+                <button mat-raised-button color="primary" (click)="addLabelComment()" *ngIf="this.labelComment != ''">
```

Comment:
Should be `!==` :)

@Darkoon - is it fine to compare strings like this in the if-statement in template? Maybe ngShow would be better here?